java-plugin/pom.xml

```diff
@@ -16,21 +16,24 @@
     <url>https://github.com/green-code-initiative/ecoCode/tree/main/java-plugin</url>
 
     <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>ecocode-rules-specifications</artifactId>
+            <version>${project.version}</version>
+            <classifier>java</classifier>
+        </dependency>
 
         <dependency>
             <groupId>org.sonarsource.java</groupId>
             <artifactId>sonar-java-plugin</artifactId>
             <type>sonar-plugin</type>
+            <scope>provided</scope>
```

Comment:
why provided ? did you test java plugin without this dependency with real java test-project ?